src/chains/ethereum/ethereum/tests/api/debug/debug_storageRangeAt.test.ts

```diff
@@ -2,33 +2,76 @@ import assert from "assert";
 import { EthereumProvider } from "../../../src/provider";
 import getProvider from "../../helpers/getProvider";
 import compile from "../../helpers/compile";
-import path from "path";
+import { join } from "path";
```

Comment:
The previous tests had tons of duplication, so I tried cleaning it up a bit. Hopefully its not worse.

There are (hopefully, ha) only three real changes to this file, and I've left a review comment on all three:

1. edited an existing test to test for the right thing
2. added a test to make sure the forking error is tested.
3. changed a static value in a contract, which changed the expected value in the test.